### DISCUSSION TITLE
Twitter like speed dial

### DISCUSSION THREAD

**Discussion by azihsoyn** [darioielardi/flutter_speed_dial#223] [Q&A] [answered]
Hi, this is awesome plugin!

I'd like to use this to achieve a twitter-like speed dial. (main fab with label and action)

https://user-images.githubusercontent.com/1225998/132783549-ead6e109-d6cf-43f6-b882-32c3d96f4294.mp4

Please advise if there is a way to do this.

Thanks.

**Comment by prateekmedia** [comment, marked as answer]
You can make that easily except the speeddial icon animation,

Just set `renderOverlay` to `false` and don't specify the label in `SpeedDialChild`.
